sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -1810,13 +1812,23 @@ public Bound withWriteDisposition(WriteDisposition writeDisposition) {
        */
       public Bound withoutValidation() {
         return new Bound(name, jsonTableRef, tableRefFunction, jsonSchema, createDisposition,
-            writeDisposition, false, bigQueryServices);
+            writeDisposition, false, bigQueryServices, ignoreUnknownValues);
+      }
+
+      /**
+       * Returns a copy of this write transformation, but with ignoreUnknownValues set to true.
```

Comment:
> but ignoring unknown values (see [BigQuery `Jobs` Documentation`](https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.load.ignoreUnknownValues)).

Maybe something like that, for clearer documentation if a reader is not super familiar with the BigQuery API?